src/Components/CreateImageWizardV2/steps/Review/targetEnvironment.tsx

```diff
@@ -0,0 +1,247 @@
+
+import {
+  Text,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+
+import {
+  GCPAccountTypes,
+  gcpAccountToString,
+} from '../TargetEnvironment/GCP/GCPTarget';
+
+const ExpirationWarning = () => {
+  return (
+    <div className="pf-u-mr-sm pf-u-font-size-sm pf-u-warning-color-100">
+      <ExclamationTriangleIcon /> Expires 14 days after creation
+    </div>
+  );
+};
+
+type TargetEnvAWSListPropTypes = {
+  manual: boolean;
+  accountId: string;
+  source: [number, string];
+};
+
+export const TargetEnvAWSList = ({
+  manual,
+  accountId,
+  source,
+}: TargetEnvAWSListPropTypes) => {
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>AWS</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Shared to account
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {accountId}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          {!manual ? 'Source' : null}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {!manual && source[1]}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Default region
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          us-east-1
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+type TargetEnvGCPListPropTypes = {
+  accountType: GCPAccountTypes;
+  accountEmail: string;
+  domain: string;
+};
+
+export const TargetEnvGCPList = ({
+  accountType,
+  accountEmail,
+  domain,
+}: TargetEnvGCPListPropTypes) => {
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>GCP</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        {(accountEmail || domain) && (
+          <>
+            <TextListItem component={TextListItemVariants.dt}>
+              Shared with
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              A Google account
+            </TextListItem>
```

Comment:
We could leave this out. It makes sense to show the "Shared with Red Hat Insight Only", but when an account is selected, there is other info that can be displayed.

Reply:
It felt more coherent to add something that the account is shared with a google account. I can remove that if you want.